DokuwikiXMLExport.php

```diff
@@ -5,9 +5,8 @@
  * If any bugs occur, please submit a new issue
  * @see https://github.com/findologic/dokuwiki-plugin-findologic-xml-export/issues/new
  */
-if (!defined('DOKU_INC')) {
-    define('DOKU_INC', realpath(dirname(__FILE__) . '/../../../') . '/');
-}
+
+if (!defined('DOKU_INC')) define('DOKU_INC', realpath(dirname(__FILE__) . '/../../') . '/');
```

Comment:
Never use conditionals without blocks (`{ ... }`). At some point you will add another statement and wonder why it does not work.

Reply:
Done.